build.sbt

```diff
@@ -34,7 +34,7 @@ libraryDependencies ++= Seq(
   "org.slf4j" % "slf4j-nop" % "1.6.4",
   "com.typesafe.play" %% "play-slick" % "1.1.1",
   "com.typesafe.play" %% "play-slick-evolutions" % "1.1.1",
-  "org.postgresql" % "postgresql" % "9.3-1102-jdbc4",
+//  "org.postgresql" % "postgresql" % "9.3-1102-jdbc4",
```

Comment:
@misaugstad commenting this out but we can actually remove this altogether. this isn't being used as the driver comes from the `slick-pg` library.

Reply:
ooooo that makes sense